docs/reference/query-dsl/exists-query.asciidoc

```diff
@@ -41,6 +41,9 @@ These documents would *not* match the above filter:
 <2> At least one non-`null` value is required.
 <3> The `user` field is missing completely.
 
+The scores produced by this query should typically not be relied upon. It is
```

Comment:
Why the confusing warning? tf values are always 1, so in the common case it just adds some delta to every document.